CMakeLists.txt

```diff
@@ -39,12 +39,20 @@ find_package(OpenCV REQUIRED)
 # catkin_python_setup()
 
 ## Setup ROSlint with our preferred settings
-## C++:
-set(ROSLINT_CPP_OPTS "--filter=-legal,-whitespace/comments,-whitespace/parens,-build/namespaces,-readability/namespace,-runtime/threadsafe_fn" "--linelength=80")
-roslint_cpp()
+## C++ Using the custom linter to avoid the ROS overloads:
+
+# Recursively select c++ source files
+file (GLOB_RECURSE FILES *.cpp *.h *.hpp) #Set the files to parse
+
+# Run the custom linter
+# To change the options for the CPP linter edit the CPPLINT.cfg file int the
+#     src directory
+roslint_custom(${CMAKE_CURRENT_LIST_DIR}/tools/linters/cpplint "--extensions=cpp,hpp,h" ${FILES})
 
 ## Python
-set(ROSLINT_PYTHON_OPTS "--ignore=E3,E121,E123,E126,E226,E24,E704" "--max-line-length=80")
+# To change the options for the python linter edit the PYLINT.cfg file in the
+#     tools/linters directory
```

Comment:
Why is the CPPLINT.cfg file in the src directory and the PYLINT.cfg file in the linters directory? For consistency, probably either move the CPPLINT.cfg file to the linters directory or the PYLINT.cfg file to the scripts directory.